viewing_party/party.py

```diff
@@ -1,23 +1,148 @@
 # ------------- WAVE 1 --------------------
 
+from symbol import subscriptlist
+
+
 def create_movie(title, genre, rating):
-    pass
+  new_move = {"title": title,
+              "genre" : genre,
+              "rating": rating}
+  for key, value in new_move.items():
+    if value == None:
+      return None
+  else:
+      return new_move
+
+def add_to_watched(user_data, movie):
+  user_data = {"watched":[movie]}
+  return user_data
+
+def add_to_watchlist(user_data,movie):
+  user_data = {"watchlist":[movie]}
+  return user_data

```

Comment:
Same comment as above, we want to avoid completely overwriting user_data. 

This function can use the append method to add movie to the watchlist